backend/app/api/routes/dde.py

```diff
@@ -243,7 +246,7 @@ async def extract_from_file(*, session: SessionDep, current_user: CurrentUser, n
             
         ).model_dump(exclude_unset=True)
     
-    chat_completion_response = await ArenaHandler(session, current_user, chat_completion_request).process_request()
+    chat_completion_response = await ArenaHandler(session, document_data_extractor.owner, chat_completion_request).process_request()
```

Comment:
do we really want to change this or is it an error?

Reply:
Yes, it enables any active user to use the endpoint even without a token